### ISSUE TITLE
menu should toogle on resize

### ISSUE THREAD

**Issue by BO41** [Chocobozzz/PeerTube#1168]
* **PeerTube version or URL:** v1.0.0-beta.16
* **Browser name/version:** firefox 62

* **What is the expected behaviour?**
the left side menu should toggle when shrinking the browser
* **What do you see instead?**
the menu stays open and the mobil css kicks in. therefore, I can't see the video anymore. and I would need to toggle it myself

![screenshot_2018-10-02 peertube demonstration 1](https://user-images.githubusercontent.com/29450609/46355462-1f0eef80-c661-11e8-9f3c-44dca8e315fd.png)

also, at some point, the interface is bugged (800px - 400px in width)

![screenshot_2018-10-02 peertube demonstration](https://user-images.githubusercontent.com/29450609/46355391-01418a80-c661-11e8-855f-a13ba9af41e1.png)

**Comment by FlatEarthTruther** [comment]
#1172 fixes the overlap issue

**Comment by Chocobozzz** [comment]
> the menu stays open and the mobil css kicks in. therefore, I can't see the video anymore. and I would need to toggle it myself

Yes, you need to close the menu manually. See https://github.com/Chocobozzz/PeerTube/pull/1172#issuecomment-426557024

We need to listen the resize event and close automatically the menu when the width < 800px.
